package.json

```diff
@@ -37,15 +37,15 @@
     "@types/node": "^10.0.0",
     "@types/node-fetch": "^2.5.7",
     "@types/puppeteer": "^3.0.1",
-    "@typescript-eslint/eslint-plugin": "^3.0.2",
-    "@typescript-eslint/parser": "^3.0.2",
+    "@typescript-eslint/eslint-plugin": "^4.3.0",
```

Comment:
it looks like these upgrades don't actually change anything in the code currently being checked. I went ahead and pushed the upgrades to separate them out and make it a bit easier to review